README.md

```diff
@@ -35,3 +35,95 @@ console.log(binaryPacket.toString('hex'))
 
 const jsonPacket = packet.deserializeIlpPayment(binaryPacket)
 ```
+
+### Serialize/deserialize ILQP Quote Requests/Responses
+
+#### QuoteLiquidityRequest
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityRequest({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  destinationAccount: 'example.nexus.bob',
+  destinationHoldDuration: 3000
+})
+
+const json = packet.deserializeIlqpLiquidityRequest(binary)
+```
+
+#### QuoteLiquidityResponse
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityResponse({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  liquidityCurve: Buffer.alloc(16), // Must be a buffer of size (n * 16) bytes
```

Comment:
The current format (array of pairs of numbers) doesn't work because JavaScript numbers don't have 64-bit precision. So we're going to get a lot of precision related bugs unless we change the format. I believe the best format to use in JS is a UInt32Array and then just write some functions that do math on curves. These functions should live in five-bells-routing.

For anyone other than five-bells-routing, liquidity curves should be an opaque object, i.e. you don't know what its internal representation is.

However, since we don't want ilp-packet to have a dependency on ilp-routing, we return a Buffer here, which is the most natural input format for a `LiquidityCurve` constructor in ilp-routing. (There is an argument that a UInt32Array would work as a neutral exchange format also, but buffers seem to be more widely used for now.)

We can first work towards making sure nobody outside of five-bells-routing tries to access the contents of a liquidity curve directly. Then, we can change the internal representation from an array to a UInt32Array in ilp-routing without breaking any other modules.